nautobot/utilities/filters.py

```diff
@@ -560,23 +553,22 @@ def _get_filter_lookup_dict(existing_filter):
         ):
             lookup_map = FILTER_NUMERIC_BASED_LOOKUP_MAP
 
-        elif isinstance(existing_filter, (TreeNodeMultipleChoiceFilter,)):
-            # TreeNodeMultipleChoiceFilter only support negation but must maintain the `in` lookup expression
-            lookup_map = FILTER_TREENODE_NEGATION_LOOKUP_MAP
-
         # These filter types support only negation
         elif isinstance(
             existing_filter,
             (
                 django_filters.ModelChoiceFilter,
                 django_filters.ModelMultipleChoiceFilter,
                 TagFilter,
+                TreeNodeMultipleChoiceFilter,
```

Comment:
What's the functional impact of this change?

Reply:
It's that these filters aren't special and `in` was never the correct lookup expression here. Multiple choice filters are handled like a list of `exact` filters.

See my comment here: https://github.com/nautobot/nautobot/issues/2513#issuecomment-1287282263

Reply:
I still don't get why the `TreeNodeMultipleChoiceFilterTest.test_filter_*_exclude` tests are passing regardless of this change from `in` to `exact` for e.g. `region__n` filters, but I'll take the fact that they *are* still passing as evidence that you're correct here. :-)

Reply:
`TreeNodeMultipleChoiceFilter.filter()` was returning a list of querysets so [`super().filter()`](https://github.com/carltongibson/django-filter/blob/2c81768188cd4ce65d5fd20919ea4a8b2b5f214b/django_filters/filters.py#L243) was building

`Q(location__in=(qs1)) | Q(location__in=(qs2) ...`

Now it's returning a list of instances so it needs to build

`Q(location=(location1)) | Q(location=(location2)) ...`

Reply:
It's more that the lookup expression matters less so long as the behavior of the filter is aligned with the filter predicate when processing a multi-choice filter vs. a single filter.

That's really the only fundamental difference between `ChoiceFilter` and `MultipleChoiceFilter`, which is the `filter()` method iterates over a list of values.

Reply:
That's what I said! It was broken in 2 places so it would work while still being broken 😆

Reply:
See also: https://django-filter.readthedocs.io/en/stable/ref/filters.html#baseinfilter